common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
@catong yes... grammar... I forgot about that.  I like your suggested re-write. Do you really think "text" is necessary, or can we do without?  I'm good either way.